requirements.txt

```diff
@@ -0,0 +1,14 @@
+# These are packages both on conda and pip
```

Comment:
I'm trying to understand in what use case I need this. If it's to *run* jax, I don't need this. I just `pip install jax jaxlib`, and the dependencies of `jax` are already specified in `setup.py`.

If it's to *test* jax, then I usually want a few extra optional dependencies (e.g., `pytest-xdist`, but I'm not required to do so.)

Should this be `requirements-for-testing.txt` or similar?

Reply:
I have made a `requirements_testing.txt` for building and testing. This is used now by Travis, readthedocs, developer build-from-source instructions. Once this gets merge I will change also the Kokoro builds.

Reply:
@hawkinsp PTAL